package.json

```diff
@@ -10,16 +10,21 @@
     "apollo-client": "^2.6.8",
     "apollo-link-http": "^1.5.16",
     "bootstrap": "^4.4.1",
+    "eslint-plugin-flowtype": "^3.13.0",
```

Comment:
Will you use Flow in this project? What is this package for?

Reply:
The thing is I already had some warnings because of one peer dependency, but when I added Formik and Yup, I saw that there 2 packages had security vulnerabilities. After I ran 'npm audit fix'  to update those, now there where 3 peer dependencies and 10 other warnings. I thought that they were too many, so I instaled the peer dependencies for react-bootstrap (jquery), apollo/react-hooks (typescript) and Yup (eslint-plugin-flowtype). Once I installed that dependencies, the other warnings for Formik and some from React where automatically gone.
I don´t know why those packages where requiered, but since it fixed all the remaining warnings I thought that it was OK to leave it like that.
But if you think that those packages are not needed, I could uninstall them and leave the warnings there.

Reply:
I tried to answer to your general comment, but I don´t know how and make some mistakes, sorry!

Reply:
It's ok to make mistakes. My comment was meant to make you think about what a package does and why would you need it. You can always search Google for a package name and read their GitHub repository to know what it's about.

Peer dependencies are dependencies needed by your dependencies that you have to install manually - **if you need a specific functionality of the package you depend on**. Let's look at two examples:
- Sequelize: you install the `sequelize` package and want to use it to access a PostgreSQL database. If you read their documentation, you'll find out that you need to install a package called `pg`. In this case `pg` is a peer dependency: you don't depend on it directly, but transitively because you want to use PostgreSQL with Sequelize.
- React-Bootstrap: just as the previous example, if you want to use the `react-bootstrap` package, you need **the CSS stylesheets** from Bootstrap. One way to include these stylesheets in your code is to install the `bootstrap` package. Now, `bootstrap` has a peer dependency of `jquery`. But that dependency is needed **only if you use the JavaScript part of the package**. In this case, you're only using the CSS part of `bootstrap` and the JavaScript from `react-bootstrap`.

Does that make sense?

Reply:
Ok, I think I understand what you mean. I thought that peer-dependencies where a must install for every package I use, but now I know that's not the case. I should only install those dependencies if I need them, and ignore them if neither I nor the package I need don't use that peer-dependency. Thanks! I will remove them and be careful to not make the same mistakes again